content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Using HPE GreenLake Console's API Gateway for Data Services Cloud Console
+date: 2021-07-29T14:28:17.779Z
+featuredBlog: false
+priority: 1
+author: Ron Dharma
+authorimage: /img/dscc-icon.png
+thumbnailimage: ""
+tags:
+  - data-services-cloud-console
+---
+## Secured yet Agile
+
+One major principle of Application Programming Interface (API) for the HPE Data Services Cloud Console (DSCC) is secured, but also flexible for consumption of multiple applications or tools that use this API to extend their function using the DSCC. One of many features for the delegated authorization below includes the ability to provide access to the API resources inside the DSCC (such as volumes, arrays, etc.) without embedding the user's credential (username, and password) in the API communication stream.
+
+### Authentication Process to Obtain the Access Token
+
+The DSCC public API relies on an OAuth 2.0 third party authorization framework on behalf of the resource owner (HPE GreenLake Console's user) for security. The user starts by logging and authenticating into HPE GreenLake Console, which is authenticated by the Identity Provider (validated through username, password, or MFA). Using the API Gateway menu in GreenLake Console, customer registers their client application to obtain the OAuth 2.0 API Client Credentials (Client ID and Client Secret). This association allows user to obtain the access-token from the menu, and user can then use the access-token into any REST API call inside the token bearer field. This action allows any client application or script to perform any HTTPS method to 
+
+The Access Token have a limited lifetime (about 720 seconds or 2 hours). Once it expires, the client application must use the obtained Client ID and Client Secret to obtain a new Access Token. One indication of the expiration of Access Token, the API will return response Error: '401 Unauthorized HTTP.'  If the user generate new Access Token prior to it expires, then the current Access Token will be invalidated or treated as not Authorized. 
```

Comment:
Is it really singular? "The Access Token"? If so, then the verb should be "has". But maybe it's supposed to be "The Access Tokens have..." With the words that follow "Once it expires", though, it would indicate that it is singular and you should change "have" to "has".

I'd recommend rephrasing "One indication of the expiration of Access Token..." to "One indication that the Access Token has expired is that the API will return a response Error:..."

Next sentence, change "generate" to "generates" - some other changes required as well. See below:
"If the user generates a new Access Token prior to its expiration, then the current..."